src/libprojectM/Renderer/MilkdropNoise.cpp

```diff
@@ -62,7 +151,7 @@ void MilkdropNoise::generate2D(int size, int zoomFactor, uint32_t** textureData)
     // smoothing
     if (zoomFactor > 1)
     {
-        dst = *textureData;
+        dst = textureData.data();
```

Comment:
This is redundant, you already assign dst above.

Reply:
No, it's not.
If you look closely, you'll notice that `dst`is being incremented in the forst loop. This assignment resets it to the start of the buffer. Without this line, projectM will just crash.

Reply:
Aha, I missed the increment. You're right, my bad!